README.md

```diff
@@ -285,8 +287,20 @@ if(context.custom.source === 'serverless-plugin-warmup'){
 
 ...
 ```
+If you're using the `concurrency` option you might consider adding a slight delay before returning when warming up to ensure your function doesn't return before all concurrent requests have been started:
 
+```javascript
+module.exports.lambdaToWarm = function(event, context, callback) {
+  /** Slightly delayed response for WarmUP plugin to ensure concurrent invocation */
```

Comment:
Can you move this comment inside the if, right before the await and rephrase to `Slightly delayed (25ms) response for WarmUP plugin to ensure concurrent invocation`